.github/workflows/scorecard.yml

```diff
@@ -8,7 +8,8 @@ on:
   workflow_dispatch:
 
 # Declare default permissions as read only.
-permissions: read-all
+permissions:
+  contents: read
```

Comment:
iIRC: scorecard wants read-all here since we have contents:read below already, but we can just try